### PR TITLE
docs: created a new documentation for openshift

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+    Your worker nodes should have additional storage disks attached. The additional storage disks should not be mounted or contain a filesystem.
+
+- **Enable Huge Pages**
+    
+    2MiB-sized Huge Pages must be supported and enabled on the storage nodes i.e. nodes where IO engine pods are deployed. A minimum number of 1024 such pages (i.e. 2GiB total) must be available exclusively to the IO engine pod on each node.
+    Huge pages in the OpenShift Container Platform (OCP) can be enabled during the installation or it can be enabled by creating new machine configs post-installation. Refer to the [Red Hat Documentation](https://access.redhat.com/solutions/5214791) for more details.
+
+- **Kernel Modules**
+
+    nvme modules are loaded by default in coreOS.
```
Comment (avishnu) [inline docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:48]: coreOS ?

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md [hunk 2]
```diff
@@ -0,0 +1,563 @@
+---
+id: openshift
+title: Replicated PV Mayastor Installation on OpenShift
```
Comment (tiagolobocastro) [inline docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:3]: Is this installation of only replicated pv mayastor? Do we want to do this or should it be OpenEBS umbrella?

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+    ```
+
+## Install Replicated PV Mayastor on OpenShift
+
+Refer to the [OpenEBS Installation Documentation](../../quickstart-guide/installation.md#installation-via-helm) to install Replicated PV Mayastor using Helm.
+
+- **Helm Install Command**
+
+```
+helm install openebs --namespace openebs openebs/openebs --create-namespace --set openebs-crds.csi.volumeSnapshots.enabled=false
```
Comment (tiagolobocastro) [inline docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:79]: This is installing the umbrella and not just replicated

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md [hunk 4]
```diff
@@ -0,0 +1,563 @@
...
+```
+
+**Output**
+
+```
+NAME   READY   STATUS    RESTARTS   AGE
+fio    1/1     Running   0          34s
+```
+
+Execute the Fio-tester application: The FIO Test utility is now run against the Replicated PV Mayastor for 60 seconds to verify that I/O operations are handled as expected and without errors. In this example, the test uses a pattern of random reads and writes, with a block size of 4KB, and a queue depth of 16.
```
Comment (tiagolobocastro) [inline docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:263]: Do we need to show this here or should we leave it at the pod is running?
This feels quite low-level output to put in a user guide, not sure if it adds any value for a user CC @avishnu

Comment (avishnu) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:263]: Hmm ya, no need for this.

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md (hunk 2 above)
Comment (avishnu) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:3]: Usually, we have mentioned Mayastor in the other guides as well.

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md (hunk 3 above)
Comment (avishnu) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:79]: Yes, but we haven't really exposed the options to disable lvm/zfs.

Comment (tiagolobocastro) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:79]: We have: https://github.com/openebs/openebs/blob/065d171cd3ac13ac334879600cc69ce4388889b1/charts/values.yaml#L56

Comment (avishnu) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:79]: Yes, we have. I'm saying we have not consciously exposed these options.

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md (hunk 2 above)
Comment (tiagolobocastro) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:3]: Should we be doing that? If we keep it engine generic then it works for every one.

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md (hunk 1 above)
Comment (balaharish7) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:48]: CoreOS in this context refers to the operating system that is optimized for containerized workloads in a Kubernetes or OpenShift environment.

File: docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md (hunk 4 above)
Comment (avishnu) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md:263]: @balaharish7 I think @tiagolobocastro is suggesting not to shw the entire output.

Comment (balaharish7) [reply docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md]: I've removed the entire output from "Run Fio-tester application"

**Review by avishnu — APPROVED** [review]

**Pull request merged**